### PR TITLE
Add a configuration number getter

### PR DESCRIPTION
**Pull request by cryptoAtwill**
Add a configuration numbers getter in the top down facet. These numbers track the configuration number applied in the child gateway.

### CODE REVIEW

**Review by snissn — APPROVED** [review]
This change makes sense to me! The addition of external view functions looks correct and valid. It would be nice to see a test for the additional getters functions. If we are currently pressed for time, could we at least document what specific tests you would like to see in the future as we merge this PR? Thanks! Looks good to me

File: contracts/src/gateway/GatewayGetterFacet.sol [hunk 1]
```diff
@@ -20,6 +21,12 @@ contract GatewayGetterFacet {
     using EnumerableSet for EnumerableSet.AddressSet;
     using EnumerableSet for EnumerableSet.Bytes32Set;
 
+    /// @notice Get the configuration numbers in the validator changes. The configuration numbers
```
Comment (jsoares) [inline contracts/src/gateway/GatewayGetterFacet.sol:24]:
```suggestion
    /// @notice Returns the configuration numbers in the validator changes. The configuration numbers
```

Nit, just for consistency.

File: contracts/src/gateway/router/TopDownFinalityFacet.sol [hunk 2]
```diff
@@ -35,6 +35,15 @@ contract TopDownFinalityFacet is GatewayActorModifiers {
         s.validatorsTracker.batchStoreChange(changeRequests);
     }
 
+    /// @notice Get the configuration numbers in the validatorTracker struct that shadows the configuration changes
```
Comment (jsoares) [inline contracts/src/gateway/router/TopDownFinalityFacet.sol:38]: Nit: the docs leak internal structure and don't actually state/explain the return values

**Review by jsoares — APPROVED** [review]

File: contracts/src/gateway/GatewayGetterFacet.sol [hunk 3]
```diff
@@ -20,6 +21,12 @@ contract GatewayGetterFacet {
...
+    /// obtained are from changes made in the parent.
```
Comment (jsoares) [inline contracts/src/gateway/GatewayGetterFacet.sol:25]:
```suggestion
    /// @notice Returns the next and start configuration numbers in the validator changes. 
    /// The configuration numbers are from changes made in the parent.
```

Nit: consistency and clarity

File: contracts/src/gateway/router/TopDownFinalityFacet.sol [hunk 4]
```diff
@@ -35,6 +35,15 @@ contract TopDownFinalityFacet is GatewayActorModifiers {
...
+    /// from the parent in the child gateway
```
Comment (jsoares) [inline contracts/src/gateway/router/TopDownFinalityFacet.sol:39]:
```suggestion
    /// @notice Get the next and start configuration numbers in the tracker of changes
    /// from the parent in the child gateway
```

Nit: clarity (explain 2 return values), abstraction (avoid leaking internal structure). It's still not super clear but probably makes sense in context.

**Pull request merged**